src/config-schema.ts

```diff
@@ -123,9 +123,24 @@ export const configSchema = {
   },
   enableStockDispense: {
     _type: Type.Boolean,
-    _description: 'Enable or disable stock deduction during the dispensing process. Requires the stock management module to be installed and configured.',
+    _description:
+      'Enable or disable stock deduction during the dispensing process. Requires the stock management module to be installed and configured.',
     _default: false,
-},
+  },
+  endVisitOnDispense: {
+    enabled: {
+      _type: Type.Boolean,
+      _description:
+        'Enables or disables the ending of the current visit upon medication dispensing. When set to true, the system will attempt to end the visit after a successful medication dispense, subject to the conditions specified in the "visitTypes" config.',
```

Comment:
Minor note, you might want to clarify that this renders a checkbox to close the visit, it doesn't automatically close the visit (if I'm understanding correctly)

Reply:
+1